elasticapm/instrumentation/register.py

```diff
@@ -69,6 +69,7 @@
     "elasticapm.instrumentation.packages.kafka.KafkaInstrumentation",
     "elasticapm.instrumentation.packages.grpc.GRPCClientInstrumentation",
     "elasticapm.instrumentation.packages.grpc.GRPCServerInstrumentation",
+    "elasticapm.instrumentation.packages.flask.FlaskInstrumentation",
```

Comment:
Seems like a reasonable solution.

I did think about this, and it did cause some of my annoying testing issues. My thought was that to run into any issue without the wrapper script, they would have to create a `Client` and call `instrument()` before creating a Flask object. Which would be pretty far from our documented usage.

Even late-initializing with [`init_app()`](https://github.com/elastic/apm-agent-python/blob/43b3554a63a581961e35bab281285f8464c53765/elasticapm/contrib/flask/__init__.py#L117) should work fine because we don't call `instrument()` until we `init_app()`.

So I think this PR is fine as written. But it may be worth splitting them out now to prevent issues with other frameworks, as well as to make testing easier. I'll look into this.